Framework/Core/src/AODReaderHelpers.cxx

```diff
@@ -176,7 +176,9 @@ AlgorithmSpec AODReaderHelpers::aodSpawnerCallback(std::vector<InputSpec>& reque
         } else if (description == header::DataDescription{"TRACKCOV_IU"}) {
           outputs.adopt(Output{origin, description, version}, maker(o2::aod::TracksCovIUExtensionMetadata{}));
         } else if (description == header::DataDescription{"TRACKEXTRA"}) {
-          outputs.adopt(Output{origin, description, version}, maker(o2::aod::TracksExtraExtensionMetadata{}));
+          outputs.adopt(Output{origin, description, version}, maker(o2::aod::TracksExtra_000ExtensionMetadata{}));
+        } else if (description == header::DataDescription{"TRACKEXTRA_001"}) {
```

Comment:
This does not seem correct, the version is encoded in subspec, not in description. I see that this is also the case for the mc particles table, which is also incorrect, it just works by accident because the extensions are identical. I will fix the case with mc particles, but here you should do the following
```cpp
if (description == header::DataDescription{"TRACKEXTRA"}) {
    if (version == 0U) {
    } else if (version == 1U) {
    }
}
```